examples/datastreams/main.go

```diff
@@ -139,11 +148,13 @@ func main() {
 		writer.Write(chunk, &onDone)
 	}
 
-	// fileName := "test.pdf"
-	// senderRoom.LocalParticipant.SendFile(fileName, lksdk.StreamBytesOptions{
-	// 	Topic:    "file-receive",
-	// 	FileName: &fileName,
-	// })
+	downloadTestFile()
+
+	topic = "file-read-all"
+	senderRoom.LocalParticipant.SendFile(testFileName, lksdk.StreamBytesOptions{
+		Topic:    topic,
+		FileName: &testFileName,
+	})
 
 	time.Sleep(30 * time.Second)
```

Comment:
Maybe use a wait group to ensure that all the stuff and exit rather than sleep?

Reply:
Right yea, I missed that while cleaning up. Let me do that.